src/Controller/Api/GalleryController.php

```diff
@@ -339,8 +339,8 @@ public function putGalleryMediaGalleryhasmediaAction($galleryId, $mediaId, Reque
      *  }
      * )
      *
-     * @param int $galleryId A gallery identifier
-     * @param int $mediaId   A media identifier
+     * @param int $galleryId Gallery identifier
```

Comment:
remove int?

Reply:
I only revert change from https://github.com/sonata-project/SonataMediaBundle/pull/1767.

Reply:
I think we should at least be consistent on this file, are we leaving the int or removing it? And the same for the property description, If we are leaving it, dont remove it for some cases

Reply:
Some bundles must be release one by one to allow use `FOS/RestBundle` becouse of dependencies. Upgrade(and fix) RestApi consists from spets like add `fos/rest-bundle` v3 support,  `nelmio/api-doc-bundle` v3 support, add support for string identifier (like UUID), functional test. It is hard to do it in one PR(and shouldn't be do it). In this release we should add support for `fos/rest-bundle` v3 and do nothing else (like add, fix or remove other docblock). I mean this should be trait as two indepentend things. The goal will be support `RestBundle` and `ApiDocBundle` v2 and v3, and support for diffrence identifier(int, UUID, string ....). I know it is not good but keep it not working it not too.